packages/icons/src/NFTs.tsx

```diff
@@ -4,7 +4,15 @@ import React from 'react';
 import NFTsIcon from './images/NFTs.svg';
 import NFTsSmallIcon from './images/NFTsSmall.svg';
 import CopyIcon from './images/copy.svg';
+import FilterIcon from './images/filter.svg';
```

Comment:
Can we use icons from material-ui instead?
Did you check if you can change color of icons properly?